routes/snippets.js

```diff
@@ -54,4 +59,56 @@ router.get("/:id", async (req, res, next) => {
   }
 });
 
+router.delete("/:id", async (req, res, next) => {
+  const { id: snippetId } = req.params;
+
+  const { auth: token } = req.cookies;
```

Comment:
개행을 생략해도 무방할 것 같습니다!

```suggestion
  const { id: snippetId } = req.params;
  const { auth: token } = req.cookies;
```